src/pages/index.tsx

```diff
@@ -241,38 +243,56 @@ const Home: NextPage<HomePageProps> = ({ storefront }) => {
             </div>
           </form>
         </div>
-        <div className="grow">
+        <div className='grow'>
           {nfts.loading ? (
             <>Loading</>
           ) : (
-            <ul className="grid grid-cols-4 gap-6">
+            <div className='grid items-center grid-cols-4 gap-2'>
               {nfts.data?.nfts.map(n => (
-                <li key={n.address}>
-                  <div className="p-4 h-68 overflow-clip hover:bg-gray">
-                    <Link
-                      href={`/nfts/${n.address}`}
-                    >
-                    <a>
-                      <img
-                        src={n.image as string}
-                        alt="nft image"
-                        className="object-fill h-56 pb-2 rounded-lg"
-                      />
-                      <div className="grid grid-cols-2 gap-2">
-                        <div>
-                          <p>{n.name}</p>
+                <div key={n.address} className='h-[450px]'>
+                  <Link href={`/nfts/${n.address}`}>
+                    <div className='h-full p-4 m-0 transition duration-500 transform hover:scale-[1.02]' >
+                      <a>
+                        <div className='w-full border border-b-0 border-[#262626] h-[250px] overflow-clip'>
```

Comment:
For the other cards we use a css trick to get the div to be a perfect square using after selector and margin. Once the box is in place and always based on the width. The image set with width height 100% and image-fit cover to position the center center cropped.